scripts/run_e2e_tests.sh

```diff
@@ -170,11 +163,7 @@ done
 # in at all.
 # TODO(bhenning): Figure out if this is a bug with protractor.
 if [ "$SHARDING" = "false" ] || [ "$SHARD_INSTANCES" = "1" ]; then
-<<<<<<< HEAD
-  $NODE_MODULE_DIR/.bin/protractor core/tests/protractor.conf.js "$SPECS"
-=======
-  $NODE_MODULE_DIR/.bin/protractor core/tests/protractor.conf.js --suite "$SUITE"
->>>>>>> develop
+  $NODE_MODULE_DIR/.bin/protractor core/tests/protractor.conf.js "$SPECS" --suite "$SUITE"
```

Comment:
This is interesting. What happens if you pass in both a spec file and a suite? What happens if you pass in an empty suite? Does all this still behave correctly?